### ISSUE TITLE
Documentation: webpack.babel and graphql-js

### ISSUE THREAD

**Issue by cahnory** [graphcool/babel-plugin-react-relay#12]
I add difficulties to make it work under these conditions:
- es6 webpack configuration file (webpack.babel.js)
- graphql-js as configuration option

getting the error: **"Schema must be an instance of GraphQLSchema"** (schemaSource was always equal to `{}` in https://github.com/graphcool/graphql-config/blob/master/src/index.ts#L98)

I had to remove _react-relay_ plugin from _.babelrc_:

``` json
{
  "presets": [
    "es2015",
    "stage-0",
    "react"
  ]
}
```

and put it back in _webpack.babel.js_, in the babel options of the webpack loader:

``` json
{
  "presets": [
    "es2015",
    "stage-0",
    "react"
  ],
  "plugins": ["react-relay"]
}
```

Maybe it's obvious but it wasn't to me. It would ne nice to have this referenced somewhere.

**Comment by schickling** [comment]
Good point. Would you'd like to create a PR with these hints in the README?

**Comment by cahnory** [comment]
I thought about it but… let's say I'm flattered that you didn't notice that my english skills are very low ^^.

**Comment by schickling** [comment]
No worries, just go for it. 👍

**Comment by cahnory** [comment]
I'll make some tests with different configuration options to see if it's really related to _grapqhl-js_ and "je me jette à l'eau" ;)
